documentation/modules/adding-source-provider.adoc

```diff
@@ -47,7 +47,7 @@ ifdef::rhv[]
 * *Hostname or IP address*: {manager} host name or IP address
 * *Username*: {manager} user
 * *Password*: {manager} password
-* *CA certificate*: CA certificate of the {manager}. If the {manager} Apache CA certificate was replaced with a third-party CA certificate, both CA certificates need to be specified.
+* *CA certificate*: Manager CA certificate unless it was replaced by a third-party certificate, in which case, specify the Manager Apache CA certificate.
```

Comment:
{manager} switches between ovirt-engine (on upstream) and RHVM (on downstream)? if so, we better keep using it ({manager})

Reply:
@ahadas Good catch; my mistake. Will fix.